### ISSUE TITLE
Unable to return values for "spans" in traces

### ISSUE THREAD

**Issue by mbonsack** [openobserve/openobserve#993]
### Which OpenObserve functionalities are the source of the bug?

traces

### Is this a regression?

No

### Description

When running the simple instrumented python example in the [docs](https://openobserve.ai/docs/ingestion/traces/python/), I cannot get any traces to expand.  This is because "spans" is always zero.  Furthermore, when I try to see the values for "spans" in the left hand fields pane, I get a popup "Error while fetching values for spans".  Logs are attached.
<img width="1362" alt="trace_span_error" src="https://github.com/openobserve/openobserve/assets/20247946/1309cd5d-c692-4810-96b9-6f6a134d5d22">
[openobserve_logs.txt](https://github.com/openobserve/openobserve/files/11789836/openobserve_logs.txt)

### Please provide the exception or error you saw

```true
"Error while fetching values for spans"
```

### Please provide the version you discovered this bug in (check about page for version information)

```true
v0.4.7
25a65024310083d36bbaa5e28ae7dae287794b04
2023-06-05T10:01:30Z
MacOS Darwin ARM64 (M-chip)
```

**Comment by prabhatsharma** [comment]
This has been resolved and will be available in next release. A duplicate of https://github.com/openobserve/openobserve/issues/926

**Comment by mbonsack** [comment]
Not sure it's exactly the same.  When I select one of the traces, I get 
<img width="1337" alt="image" src="https://github.com/openobserve/openobserve/assets/20247946/73d29241-80fa-4a81-9a46-f54a45b3915b">
which is not blank, but also does not show the spans.  Maybe it's related to #918, which is cited as the fix for #926?

**Comment by mbonsack** [comment]
Still happens at 0.4.8.
